Packages/com.unity.inputsystem/Documentation~/Actions.md

```diff
@@ -33,7 +33,7 @@ When scripting with Actions in the Input System, there are number of important c
 
 |Class|Description|
 |-----|-----------|
-|`InputSystem.actions`|A reference to the set of actions configured in the [Input Actions editor](ActionsEditor.md). |
+|[`InputSystem.actions`](../api/UnityEngine.InputSystem.InputSystem.html)|A reference to the set of actions configured in the [Input Actions editor](ActionsEditor.md). |
```

Comment:
Should this tie back to the project wide aspect? Its currently a bit generic, e.g. 
"A reference to the set of project-wide actions configured in...."

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)